js/main.js

```diff
@@ -639,18 +733,45 @@ function updateBodyClasses() {
     document.getElementById('body').classList.toggle('game-playing', isPlaying());
 }
 
+function doTasks() {
+    /*
+    const modules = gameData.activeModules;
+    if (modules == null) return;
+    for (const moduleName in modules){
+        const module = modules[moduleName];
+        module.do();
+    }*/
+
+    const tasks = gameData.currentOperations;
+    if (tasks !== null){
+        for (const taskName in tasks){
+            const task = tasks[taskName];
+            if(task != null)
+                task.do();
+        }
+    }

```

Comment:
_Könnte_ man auch einfach so bauen:
```suggestion
        tasks.forEach(function (task) { task.do(); });
```
oder richtig dope mit ES6:
```
        tasks.forEach(task => task.do());
```
wenn wir das wollen sollten wir uns mal über unsere JavaScript Version unterhalten. Browser support sollte vollkommen okay sein für ES6